lib/datadog/appsec/processor/rule_merger.rb

```diff
@@ -16,8 +18,13 @@ def initialize(version1, version2)
           end
         end
 
+        DEFAULT_PROCESSORS = JSON.parse(Datadog::AppSec::Assets.waf_processors)
```

Comment:
I'd call that `DEFAULT_WAF_PROCESSORS` to materialize the distinction with `AppSec::Processor`

Reply:
I would also refrain from reading+parsing that file + setting that constant at file load time. I'd rather have that be a memoized method.